pandas/core/strings.py

```diff
@@ -1122,21 +1127,17 @@ def str_join(arr, sep):
     Example with a list that contains non-string elements.
 
     >>> s = pd.Series([['lion', 'elephant', 'zebra'],
-    ...                [1.1, 2.2, 3.3],
-    ...                ['cat', np.nan, 'dog'],
-    ...                ['cow', 4.5, 'goat']
-    ...                ['duck', ['swan', 'fish'], 'guppy']])
+    ...            [1.1, 2.2, 3.3],
+    ...            ['cat', np.nan, 'dog'],
+    ...            ['cow', 4.5, 'goat'],
```

Comment:
Good catch of the typo, but I think the indentaion was correct before.

Reply:
D'oh! Thank you.